test/tests/meshgenerators/sidesets_from_points_generator/tests

```diff
@@ -9,5 +9,15 @@
     issues = '#11640'
     mesh_mode = 'REPLICATED'
     recover = false
-  [../]
+  []
+  [sidesets_ambiguity]
+    type = 'RunException'
+    input = 'sidesets_ambiguity.i'
+    cli_args = '--mesh-only'
+    expect_err = "ambiguous potential sideset"
+    requirement = 'The system shall return errors if it detects ambiguous sideset specifications'
+    design = 'meshgenerators/SideSetsFromPointsGenerator.md'
+    issues = '#24619'
+    recover = false
+  []
```

Comment:
```suggestion
    mesh_mode = 'replicated'
  []
```
needed for stitched meshes

Reply:
Not since last year: https://github.com/libMesh/libmesh/pull/3262